src/core/ext/transport/chaotic_good/client_transport.cc

```diff
@@ -63,6 +63,7 @@ ClientTransport::ClientTransport(
           ResourceQuota::Default()->memory_quota()->CreateMemoryAllocator(
               "client_transport")),
       arena_(MakeScopedArena(1024, &memory_allocator_)),
+      context_(arena_.get()),
```

Comment:
Is the context_ used anywhere ?

Reply:
Yes, it is used at https://source.corp.google.com/piper///depot/google3/third_party/grpc/src/core/lib/promise/context.h;rcl=572915049;l=80. Without initializing this `context_`, the error test will fail.